test/integration/targets/podman_container/tasks/main.yml

```diff
@@ -324,3 +334,12 @@
             '/tmp' in test.ansible_facts.podman_container['Mounts'] | map(attribute='Source') | list
         fail_msg: Parameters container test failed!
         success_msg: Parameters container test passed!
+
+    - name: Delete all container leftovers from tests
```

Comment:
This should probably go to `block`->`always` section https://docs.ansible.com/ansible/latest/user_guide/playbooks_blocks.html#id3